### PR TITLE
Update Russian translations of UI

### PR DESCRIPTION
**Pull request by IZOBRETATEL777**
## Description

Update translations of UI

## Checklist

- [ ] If the key of a setting is changed, the `old` attribute is updated or it is resolved in SettingsUpdater.
- [ ] If there are changes of the text displayed in the UI, they are wrapped in `tr()` or `QCoreApplication::translate()`.
- [ ] If needed, I have opened a pull request or an issue to update the [documentation](https://github.com/cpeditor/cpeditor.github.io).
- [ ] If these changes are notable, they are documented in [CHANGELOG.md](https://github.com/cpeditor/cpeditor/blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by coder3101** [comment]
Approval will be done by @sadykhzadeh if not done in 12 hours, we will approve it.

@ouuan Do you think we need to make a new minor release with updated translation?

**Comment by ouuan** [comment]
I think it's fine to have only a few text untranslated in the release.

File: translations/ru_RU.ts [hunk 1]
```diff
@@ -2658,35 +2658,35 @@ You can learn about it by running `yapf --style-help`.</source>
...
     </message>
     <message>
         <source>Donate</source>
-        <translation type="unfinished"></translation>
+        <translation>Пожертвовать</translation>
     </message>
     <message>
         <source>I&apos;m using @cpeditor_, an IDE specially designed for competitive programmers, which is awesome!</source>
-        <translation type="unfinished"></translation>
+        <translation>Я использую @cpeditor_ — специальная IDE для спортивного программирования и она отлична!</translation>
```
Comment (sadykhzadeh) [inline translations/ru_RU.ts:2689]: @IZOBRETATEL777 а это нижнее подчёркивание точно нужно?

И предлагаю вместо "отлична" слово "потрясна"

**Review by sadykhzadeh — APPROVED** [review]
It's OK.

**Comment by ouuan** [comment]
@allcontributors add @Sadykhzadeh for review.

**Pull request merged**